### PR TITLE
morph clean status tag, add a new class CleanStatusTag

### LINKED ISSUE #39: Morph clean status
replicate current 'tag' to be mandatory 'clean' field

### PR DESCRIPTION
**Pull request by wangtao0717**
fix #39. add a new clean status tag, now it can still add more than one like tag, I will fix it later

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -54,7 +57,7 @@
     private final EditPersonDescriptor editPersonDescriptor;
 
     /**
-     * @param index of the person in the filtered person list to edit
+     * @param index                of the person in the filtered person list to edit
```
Comment (awzhenyi) [inline src/main/java/seedu/address/logic/commands/EditCommand.java]: there is a large chunk of spaces(?) here, not sure if it is intended

File: src/main/java/seedu/address/logic/Logic.java [hunk 2]
```diff
@@ -16,10 +16,11 @@
 public interface Logic {
     /**
      * Executes the command and returns the result.
+     *
      * @param commandText The command as entered by the user.
      * @return the result of the command execution.
      * @throws CommandException If an error occurs during command execution.
-     * @throws ParseException If an error occurs during parsing.
+     * @throws ParseException   If an error occurs during parsing.
```
Comment (awzhenyi) [inline src/main/java/seedu/address/logic/Logic.java:23]: There is a space here, not sure if it is intended?

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 1 above)
Comment (wangtao0717) [reply src/main/java/seedu/address/logic/commands/EditCommand.java]: oh,i don't know when i space here... just delete it

File: src/main/java/seedu/address/logic/Logic.java (hunk 2 above)
Comment (wangtao0717) [reply src/main/java/seedu/address/logic/Logic.java:23]: it is for checkstyle i think

**Review by awzhenyi — APPROVED** [review]
lgtm

**Pull request merged**